sample-parent/sample-connector/src/main/java/org/talend/sdk/component/test/connectors/service/UIService.java

```diff
@@ -115,4 +124,36 @@ public ValidationResult retrieveValidation() throws Exception {
         return result;
     }
 
+    /**
+     * Healthcheck action OK
+     *
+     * Documentation: https://talend.github.io/component-runtime/main/latest/services-actions.html#_healthcheck
+     * Type: healthcheck
+     * API: @org.talend.sdk.component.api.service.healthcheck.HealthCheck
+     *
+     * Returned type: org.talend.sdk.component.api.service.healthcheck.HealthCheckStatus
+     */
+    @HealthCheck(HEALTH_CHECK_OK)
+    public HealthCheckStatus discoverHealthcheckOk(@Option final TheDatastore unused) {
+        try {
+            return new HealthCheckStatus(Status.OK, "HealthCheck Success");
+        } catch (Exception e) {
```

Comment:
I don't think that you still need the try/catch

Reply:
OK, remove the try/catch